practical-unit-testing-with-rtl/Exercise1-queries/solution/src/App.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+
+const Button = () => {
+  return <button> Click me to submit!</button>;
+};
+
+const TextInput = () => {
+  return (
+    <>
+      <label htmlFor="username">Username</label>
+      <input id="username" placeholder="Enter Username Here" />
+    </>
+  );
+};
+
+const ColorPicker = () => {
+  return (
+    <>
+      <label htmlFor="color">Select Your Favorite Color:</label>
+      <select id="color" name="color">
+        <option value="red" selected>
+          Red
+        </option>
+        <option value="blue">Blue</option>
+        <option value="green">Green</option>
+        <option value="yellow">Yellow</option>
+        <option value="purple">Purple</option>
+      </select>
+    </>
+  );
+};
+
+const App = () => {
+  return (
+    <>
+      <h1 data-testid="main-header">Welcome to our App!</h1>
```

Comment:
I’m guessing the purpose of this is to use `getByTestId()`, right?

If so, I think we should try to move this to another element, because in general we’d want folks using `getByRole('heading')` to query for this header, right?

Reply:
yeah, in general. my thoughts with this were: 
1.) this is to just get people familiar with the queries 
2.) this opens up the discussion more on "which one SHOULD be used"